server/graphql/v2/object/HostMetricsTimeSeries.ts

```diff
@@ -51,5 +51,14 @@ export const HostMetricsTimeSeries = new GraphQLObjectType({
         return { dateFrom, dateTo, timeUnit, nodes: resultsToAmountWithSettlementNode(results) };
       },
     },
+    totalMoneyManagedProgress: {
```

Comment:
..Progress is a weird name for the property.

Reply:
Change the naming to `totalMoneyManagedTimeSeries`

Reply:
Other `TimeSeries` stats don't have a suffix `TimeSeries` so I don't see why this particular stat deserve a suffix.

Reply:
@znarf : You are correct, we don't need the suffix; I've just made it `totalMoneymanaged` so that it's consistent.